app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
@niranjan94 We will be creating a model in speaker-list as it is the parent to the speaker-item(child) in this case. So it is already getting passed from the parent to its children.